.circleci/config.yml

```diff
@@ -187,7 +187,8 @@ aliases:
 
   - &run-flow-checks
     name: Check for errors in code using Flow
-    command: yarn flow check
+    command: yarn flow-check-ios
+    command: yarn flow-check-android
```

Comment:
Woah! We weren't checking this!? Scary!

Reply:
That only runs checking with the default config which checks .ios.js files but not .android.js files.